### PR TITLE
translation of the package-manager page in download section

### PR DESCRIPTION
**Pull request by maugravena**
Translated download/package-manager.md from en to pt-BR

Refs: [nodejs/nodejs-pt#72](https://github.com/nodejs/nodejs-pt/issues/72)

/cc @nodejs/nodejs-pt

### CODE REVIEW

File: locale/pt-br/download/package-manager.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+
+# Instalando Node.js via gerenciador pacotes
+
+***Nota:*** Os pacotes nessa página são mantidos e suportados por seus respectivos empacotadores, **não** pela equipe principal do Node.js. Por favor, relate quaisquer problemas encontrados para o mantenedor do pacote.
+
+----------------------------
+
+* [Android](#android)
+* [Arch Linux](#arch-linux)
+* [Debian e Destribuições Linux baseadas em Ubuntu, Enterprise Linux/Fedora e pacotes Snap](#debian-and-ubuntu-based-linux-distributions-enterprise-linux-fedora-and-snap-packages)
```
Comment (Drieger) [inline locale/pt-br/download/package-manager.md]:
```suggestion
* [Debian e Distribuições Linux baseadas em Ubuntu, Enterprise Linux/Fedora e pacotes Snap](#debian-and-ubuntu-based-linux-distributions-enterprise-linux-fedora-and-snap-packages)
```

File: locale/pt-br/download/package-manager.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+
+Ou instale um pacote binário (se estiver disponível para sua plataforma)  usando pkgin:
+
+```bash
+pkgin -y install nodejs
+```
+
+## nvm
+
+Node Version Manager é um bash script usado para gerenciar múltiplas versões lançadas do Node.js. Ele Permite
```
Comment (Drieger) [inline locale/pt-br/download/package-manager.md]:
```suggestion
Node Version Manager é um script bash utilizado para gerenciar múltiplas versões do Node.js. Ele Permite
```

File: locale/pt-br/download/package-manager.md [hunk 3]
```diff
@@ -0,0 +1,239 @@
+---
+layout: page.hbs
+title: Instalando Node.js via gerenciador pacotes
+---
+
+# Instalando Node.js via gerenciador pacotes
+
+***Nota:*** Os pacotes nessa página são mantidos e suportados por seus respectivos empacotadores, **não** pela equipe principal do Node.js. Por favor, relate quaisquer problemas encontrados para o mantenedor do pacote.
```
Comment (JCMais) [inline locale/pt-br/download/package-manager.md]: This is missing a sentence at the end:

> If it turns out your issue is a bug in Node.js itself, the maintainer will report the issue upstream.

File: locale/pt-br/download/package-manager.md [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+* [SmartOS e illumos](#smartos-and-illumos)
+* [Solus](#solus)
+* [Void Linux](#void-linux)
+* [Windows](#windows)
+
+----------------------------
+
+## Android
+
+O suporte ao Android ainda é experimental no Node.js, portanto, os binários pré-compilados ainda não são fornecidos pelos programadores do Node.js
```
Comment (JCMais) [inline locale/pt-br/download/package-manager.md]:
```suggestion
O suporte ao Android ainda é experimental no Node.js, portanto, os binários pré-compilados ainda não são fornecidos pelos desenvolvedores do Node.js
```

File: locale/pt-br/download/package-manager.md [hunk 5]
```diff
@@ -0,0 +1,239 @@
...
+
+Entretanto, existem algumas soluções de terceiros. Por exemplo, a comunidade [Termux](https://termux.com/) que fornece um emulador de terminal e ambiente Linux para Android, assim como seu próprio gerenciador de pacotes e [extensa coleção](https://github.com/termux/termux-packages) com muitas aplicações pré-compiladas. Esse comando no Termux vai instalar a última versão disponível do Node.js:
```
Comment (JCMais) [inline locale/pt-br/download/package-manager.md]:
```suggestion
Entretanto, existem algumas soluções de terceiros. Por exemplo, a comunidade [Termux](https://termux.com/) que fornece um emulador de terminal e ambiente Linux para Android, assim como seu próprio gerenciador de pacotes e [extensa coleção](https://github.com/termux/termux-packages) com muitas aplicações pré-compiladas. Esse comando vai instalar a última versão disponível do Node.js no Termux:
```

File: locale/pt-br/download/package-manager.md [hunk 6]
```diff
@@ -0,0 +1,239 @@
...
+
+```bash
+pkg install nodejs
+```
+
+Atualmente, os binários do Node.js no Termux estão ligados ao `system-icu` (dependência do pacote `libicu`)
+
+## Arch Linux
+
+O Node.js pacotes npm estão disponíveis no Repositório da Comunidade.
```
Comment (JCMais) [inline locale/pt-br/download/package-manager.md]:
```suggestion
Os pacotes para o Node.js e o npm estão disponíveis no Repositório da Comunidade.
```

File: locale/pt-br/download/package-manager.md [hunk 7]
```diff
@@ -0,0 +1,239 @@
...
+
+```bash
+pacman -S nodejs npm
+```
+
+## Debian e distribuições Linux basedas em  Ubuntu, Enterprise Linux/Fedora e pacotes Snap
```
Comment (JCMais) [inline locale/pt-br/download/package-manager.md]: Extra space before Ubuntu

File: locale/pt-br/download/package-manager.md [hunk 8]
```diff
@@ -0,0 +1,239 @@
...
+
+[Distribuição dos binários oficiais do Node.js](https://github.com/nodesource/distributions/blob/master/README.md) são fornecidos pelo NodeSource.
+
+## FreeBSD
+
+A versão mais recente do Node.js está disponível na porta [www/node](http://freshports.org/www/node).
```
Comment (JCMais) [inline locale/pt-br/download/package-manager.md:55]: The translation of bsb port to `porta` here feels weird, but I don't know any better translation

File: locale/pt-br/download/package-manager.md [hunk 9]
```diff
@@ -0,0 +1,239 @@
...
+
+Intalar um pacote binário via [pkg](https://www.freebsd.org/cgi/man.cgi?pkg):
```
Comment (JCMais) [inline locale/pt-br/download/package-manager.md]:
```suggestion
Instale um pacote binário via [pkg](https://www.freebsd.org/cgi/man.cgi?pkg):
```

File: locale/pt-br/download/package-manager.md [hunk 10]
```diff
@@ -0,0 +1,239 @@
...
+
+```bash
+pkg install node
+```
+
+Ou compile por conta própria usando [ports](https://www.freebsd.org/cgi/man.cgi?ports):
```
Comment (JCMais) [inline locale/pt-br/download/package-manager.md]: `compile-o`

File: locale/pt-br/download/package-manager.md [hunk 11]
```diff
@@ -0,0 +1,239 @@
...
+
+```bash
+cd /usr/ports/www/node && make install
+```
+
+## Gentoo
+
+O Node.js está disponível na portage tree.
```
Comment (JCMais) [inline locale/pt-br/download/package-manager.md]: You can translate `tree` as `diretório`
```suggestion
O Node.js está disponível no diretório portage.
```

File: locale/pt-br/download/package-manager.md [hunk 12]
```diff
@@ -0,0 +1,239 @@
...
+
+```bash
+emerge nodejs
+```
+
+## NetBSD
+
+Node.js está disponível na pkgsrc tree:
```
Comment (JCMais) [inline locale/pt-br/download/package-manager.md]: Same than above

File: locale/pt-br/download/package-manager.md [hunk 13]
```diff
@@ -0,0 +1,239 @@
...
+
+```bash
+cd /usr/pkgsrc/lang/nodejs && make install
+```
+
+Ou instale um pacote binário (se estiver disponível para sua plataforma)  usando pkgin:
```
Comment (JCMais) [inline locale/pt-br/download/package-manager.md]: extra space before `usando`

File: locale/pt-br/download/package-manager.md (hunk 2 above)
Comment (JCMais) [inline locale/pt-br/download/package-manager.md]: casing, `Ele Permite`

File: locale/pt-br/download/package-manager.md [hunk 14]
```diff
@@ -0,0 +1,239 @@
...
+
+```bash
+pkgin -y install nodejs
+```
+
+## nvm
+
+Node Version Manager é um bash script usado para gerenciar múltiplas versões lançadas do Node.js. Ele Permite
+que você execute operações como instalar, desinstalar e mudar de versão etc. Para instalar o nvm, use esse [script de instalação](https://github.com/creationix/nvm#install-script).
```
Comment (JCMais) [inline locale/pt-br/download/package-manager.md]:
```suggestion
que você instale, desinstale e mude de versão etc. Para instalar o nvm, use esse [script de instalação](https://github.com/creationix/nvm#install-script).
```

File: locale/pt-br/download/package-manager.md [hunk 15]
```diff
@@ -0,0 +1,239 @@
...
+
+Em sistemas Unix / OS X o Node.js construído a partir do fonte pode ser instalado usando [nvm](https://github.com/creationix/nvm), instalando no
```
Comment (JCMais) [inline locale/pt-br/download/package-manager.md]:
```suggestion
Em sistemas Unix / OS X o Node.js compilado a partir do código fonte pode ser instalado usando [nvm](https://github.com/creationix/nvm), instalando-o no
```

File: locale/pt-br/download/package-manager.md [hunk 16]
```diff
@@ -0,0 +1,239 @@
...
+local em que o nvm espera:
+
+```bash
+$ env VERSION=`python tools/getnodeversion.py` make install DESTDIR=`nvm_version_path v$VERSION` PREFIX=""
+```
+
+Despois disso, você pode usar o `nvm` para alternar entre versões lançadas e versões
+construidas a partir do fonte.
```
Comment (JCMais) [inline locale/pt-br/download/package-manager.md]:
```suggestion
compiladas a partir do código fonte.
```

File: locale/pt-br/download/package-manager.md [hunk 17]
```diff
@@ -0,0 +1,239 @@
...
+Por exemplo, se a versão do Node.js é v8.0.0-pre:
+
+```bash
+$ nvm use 8
+```
+
+Uma vez que a versão oficial for lançada você pode querer desinstalar a versão construída
```
Comment (JCMais) [inline locale/pt-br/download/package-manager.md]:
```suggestion
Uma vez que a versão oficial for lançada você pode querer desinstalar a versão compilada
```

File: locale/pt-br/download/package-manager.md [hunk 18]
```diff
@@ -0,0 +1,239 @@
...
+a partir do fonte:
```
Comment (JCMais) [inline locale/pt-br/download/package-manager.md]:
```suggestion
a partir do código fonte:
```

File: locale/pt-br/download/package-manager.md [hunk 19]
```diff
@@ -0,0 +1,239 @@
...
+
+```bash
+$ nvm uninstall 8
+```
+
+## OpenBSD
+
+O Node.js está disponível através das portas do sistema.
```
Comment (JCMais) [inline locale/pt-br/download/package-manager.md:119]: `portas do sistema` sound weird, but idk on a better translation

File: locale/pt-br/download/package-manager.md [hunk 20]
```diff
@@ -0,0 +1,239 @@
...
+
+Usando [pkg_add](http://man.openbsd.org/OpenBSD-current/man1/pkg_add.1) no OpenBSD:
+
+```bash
+pkg_add node
+```
+
+## openSUSE e SLE
+
+Node.js está disponível nos principais repositórios nos seguintes pacotes:
```
Comment (JCMais) [inline locale/pt-br/download/package-manager.md]:
```suggestion
Node.js está disponível nos principais repositórios sob os seguintes pacotes:
```

File: locale/pt-br/download/package-manager.md [hunk 21]
```diff
@@ -0,0 +1,239 @@
...
+
+## SmartOS e illumos
+
+Imagens do SmartOS vêm com o pkgsrc pré-instalado. Em outras distribuições illumos, primeiro instale **[pkgsrc](https://pkgsrc.joyent.com/install-on-illumos/)**, então você pode instalar o binário normalmente:
+
+```bash
+pkgin -y install nodejs
+```
+
+Ou construa manualmente pelo pkgsrc:
```
Comment (JCMais) [inline locale/pt-br/download/package-manager.md]:
```suggestion
Ou compile manualmente pelo pkgsrc:
```

File: locale/pt-br/download/package-manager.md [hunk 22]
```diff
@@ -0,0 +1,239 @@
...
+
+Usando **[pkgsrc](https://pkgsrc.joyent.com/install-on-osx/)**:
+
+Instale o pacote binário:
+
+```bash
+pkgin -y install nodejs
+```
+
+Ou constrindo manualmente pelo pkgsrc:
```
Comment (JCMais) [inline locale/pt-br/download/package-manager.md]:
```suggestion
Ou compile manualmente pelo pkgsrc:
```

File: locale/pt-br/download/package-manager.md [hunk 23]
```diff
@@ -0,0 +1,239 @@
...
+
+Apenas baixe o [Windows Installer](https://nodejs.org/#download) diretamente do site [nodejs.org](https://nodejs.org).
+
+### Alternativas
+
+Usando **[Chocolatey](http://chocolatey.org)**:
+
+```bash
+cinst nodejs
+# or for full install with npm
```
Comment (JCMais) [inline locale/pt-br/download/package-manager.md]: translate the comment

File: locale/pt-br/download/package-manager.md (hunk 11 above)
Comment (maugravena) [reply locale/pt-br/download/package-manager.md]: Eu estava de fato com dúvidas nesse termo

**Review by JCMais — COMMENTED** [review]
Great work overall @maugravena 

Thanks for taking the time to translate that page

**Comment by ghost** [comment]
@nodejs/nodejs-pt：Are there any other reviews?

File: locale/pt-br/download/package-manager.md [hunk 24]
```diff
@@ -0,0 +1,239 @@
...
+
+```bash
+zypper install nodejs4
+```
+
+## macOS
+
+Basta baixar o [macOS Installer](https://nodejs.org/#download) diretamente do site [nodejs.org](https://nodejs.org).
+
+_Se você quer baixar o pacote com bash:_
```
Comment (Drieger) [inline locale/pt-br/download/package-manager.md]:
```suggestion
_Se desejar baixar o pacote com bash:_
```

File: locale/pt-br/download/package-manager.md [hunk 25]
```diff
@@ -0,0 +1,239 @@
...
+O Solus fornece o Node.js em seu repositório principal.
+
+```bash
+sudo eopkg install nodejs
+```
+
+
+## Void Linux
+
+Void Linux entrega o Node.js estável no repositório principal.
```
Comment (Drieger) [inline locale/pt-br/download/package-manager.md]:
```suggestion
Void Linux possui versões estáveis de Node.js no seu repositório principal.
```

File: locale/pt-br/download/package-manager.md [hunk 26]
```diff
@@ -0,0 +1,239 @@
...
+
+```bash
+xbps-install -Sy nodejs
+```
+
+## Windows
+
+Apenas baixe o [Windows Installer](https://nodejs.org/#download) diretamente do site [nodejs.org](https://nodejs.org).
```
Comment (Drieger) [inline locale/pt-br/download/package-manager.md]:
```suggestion
Baixe o [Windows Installer](https://nodejs.org/#download) diretamente do site [nodejs.org](https://nodejs.org).
```

**Review by Drieger — COMMENTED** [review]
Thanks for the awesome work here @maugravena. I left some suggestions.

File: locale/pt-br/download/package-manager.md [hunk 27]
```diff
@@ -0,0 +1,239 @@
...
+Ou instale um pacote binário (se estiver disponível para sua plataforma)  usando pkgin:
+
+```bash
+pkgin -y install nodejs
+```
+
+## nvm
+
+Node Version Manager é um bash script usado para gerenciar múltiplas versões lançadas do Node.js. Ele Permite
+que você instale, desinstale e mude de versão etc. Para instalar o nvm, use esse [script de instalação](https://github.com/creationix/nvm#install-script).
```
Comment (Drieger) [inline locale/pt-br/download/package-manager.md]:
```suggestion
que você instale, desinstale, mude de versão e etc. Para instalar o nvm, use esse [script de instalação](https://github.com/creationix/nvm#install-script).
```

**Review by Drieger — APPROVED** [review]

**Comment by Drieger** [comment]
I think you just need to fix what is missing pointed by @JCMais

File: locale/pt-br/download/package-manager.md (hunk 3 above)
Comment (maugravena) [reply locale/pt-br/download/package-manager.md]: Não consigo pensar em uma boa tradução para "issue upstream"

**Comment by maugravena** [comment]
My pleasure, we work together @Drieger @JCMais 

There are some reviews that need to be resolved, do you guys have any suggestions?

**Comment by Drieger** [comment]
LGTM

**Comment by Maledong** [comment]
Hi @JCMais , is it going well with you?

**Pull request merged**